facia-press/app/controllers/Application.scala

```diff
@@ -1,14 +1,25 @@
 package controllers
 
+import frontpress.FrontPress
+import play.api.libs.json.Json
 import play.api.mvc.{Action, Controller}
 import services.ConfigAgent
 
 object Application extends Controller {
+  import scala.concurrent.ExecutionContext.Implicits.global
   def index = Action {
     Ok("Hello, I am the Facia Press.")
   }
-  
+
   def showCurrentConfig = Action {
     Ok(ConfigAgent.contentsAsJsonString).withHeaders("Content-Type" -> "application/json")
   }
+
+  def generateDraftPressedFor(path: String) = Action.async { request =>
```

Comment:
Unless there is a pressing (excuse the pun) need to the draft endpoint just drop it. I we really need it come talk to me today.